### ISSUE TITLE
Waybar's Hyprland/window module doesn't recognize fullscreen stacked groups.

### ISSUE THREAD

**Issue by izmyname** [hyprwm/Hyprland#8210]
### Already reported ? *

- [X] I have searched the existing open and closed issues.

### Regression?

No

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
<Hyprland 0.44.0 built from branch main at commit 6e0aadc585c6d9fdaaebfa5853adbf9610897c82  (updated flake.lock).
Date: Wed Oct 23 09:19:55 2024
Tag: v0.44.0-69-g6e0aadc5, commits: 5368
built against aquamarine 0.4.3


flags set:
debug


System Information:
System name: Linux
Node name: archbtw
Release: 6.11.5-zen1-1-zen
Version: #1 ZEN SMP PREEMPT_DYNAMIC Tue, 22 Oct 2024 18:31:33 +0000


GPU information: 
00:02.0 VGA compatible controller [0300]: Intel Corporation CometLake-H GT2 [UHD Graphics] [8086:9bc4] (rev 05) (prog-if 00 [VGA controller])
01:00.0 VGA compatible controller [0300]: NVIDIA Corporation TU116M [GeForce GTX 1660 Ti Mobile] [10de:2191] (rev a1) (prog-if 00 [VGA controller])
NVRM version: NVIDIA UNIX x86_64 Kernel Module  560.35.03  Fri Aug 16 21:39:15 UTC 2024


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:>
```

</details>

### Description

I'm sorry if this isn't Hyprland, but Waybar issue, but I'm unsure, so I'll leave this here, as well.

https://github.com/Alexays/Waybar/issues/3681

tldr: Waybar's Hyprland/window module doesn't recognize focused fullscreen stacked windowgroup as fullscreen one. Could be the problem on Hyprland's side, something about group windows aren't correctly marked as being on fullscreen ones by the compositor.

### How to reproduce

Create a stacked group and fullscreen it.
